python/lsst/pipe/base/config.py

```diff
@@ -294,6 +314,12 @@ class PipelineTaskConfig(pexConfig.Config):
     quantum = pexConfig.ConfigField(dtype=QuantumConfig,
                                     doc="configuration for PipelineTask quantum")
 
+    @_templateNameStorageDecorator
+    def formatTemplateNames(self, templateParamDict):
```

Comment:
I'd recommend renaming this to `substituteDatasetNames` for consistency with the command-line argument.